pkg/sidecar-controller/snapshot_controller.go

```diff
@@ -339,7 +339,12 @@ func (ctrl *csiSnapshotSideCarController) createSnapshotWrapper(content *crdv1.V
 		return content, fmt.Errorf("failed to add VolumeSnapshotBeingCreated annotation on the content %s: %q", content.Name, err)
 	}
 
-	parameters, err := utils.RemovePrefixedParameters(class.Parameters)
+	var parameters map[string]string
```

Comment:
I think we should not call createSnapshotWrapper() if it is a member of a dynamically provisioned volume groupsnapshot.  I need to look into more and see where is the best place to do that check.

Reply:
I'm proposing another fix. I'm trying another fix; let's see if it is more reasonable.